campair/campair/View/Editor/EditorDetailView.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  EditorDetailView.swift
+//  campair
+//
+//  Created by 이가은 on 2022/06/13.
+//
+
+import SwiftUI
+
+extension Color {
+    static let customPaintingTextPink = Color(#colorLiteral(red: 0.9008693695, green: 0.5397311449, blue: 0.4869769216, alpha: 1))
```

Comment:
```suggestion
    enum Custom {
        static let paintingTextPink = Color(#colorLiteral(red: 0.9008693695, green: 0.5397311449, blue: 0.4869769216, alpha: 1))
    }
```

enum에서 이름이 붙는다면 사용할 때 불러서 사용하기에도 편할 것 같아요 : )

Reply:
Thank you. I will study and fix it when the color is fixed.